CesiumGltf/test/TestPropertyTypeTraits.cpp

```diff
@@ -193,275 +193,643 @@ TEST_CASE("Test MetadataArrayType") {
   REQUIRE(std::is_same_v<type, uint32_t>);
 }
 
-TEST_CASE("TypeToPropertyType") {
-  SUBCASE("Works for scalar types") {
-    REQUIRE(TypeToPropertyType<uint8_t>::value == PropertyType::Scalar);
-    REQUIRE(
-        TypeToPropertyType<uint8_t>::component == PropertyComponentType::Uint8);
+TEST_CASE("Test canRepresentPropertyType") {
```

Comment:
Sorry to trouble but do you mind ordering the tests starting from `TypeToPropertyType` -> `CanBeNormalized` -> `canRepresentPropertyType`? I'm sure there's nothing wrong with the content, but the visual diff is large on Github so it's a bit hard to compare